packages/repository/src/query.ts

```diff
@@ -60,6 +61,7 @@ export type PredicateComparison<PT> = {
   ilike?: PT;
   nilike?: PT;
   regexp?: string | RegExp;
+  near?: object;
```

Comment:
Need a more specific type here to show it is a `GeoPoint` object. We have a definition in our `geo-services` example. Maybe we can extract that out to `repository` layer types. Thoughts?

Reply:
IIRC, the type used by `geo-services` is not exactly the same one as we use in LoopBack and connectors. I made the type closer to the external webservice called by the example app to make the code simpler.

Please see https://github.com/strongloop/loopback-next/issues/1981#issuecomment-436966284 and the related discussion for different options.

For now, I am proposing to use `any` or an inline type.

An example of an inline type to illustrate what I mean is below. The type definition is probably not correct, please don't take it as-is.

```ts
near?: {lat: number, lng: number} | [number, number];
```